pkg/agent/types/networkpolicy.go

```diff
@@ -20,12 +20,40 @@ import (
 	binding "github.com/vmware-tanzu/antrea/pkg/ovs/openflow"
 )
 
+type MatchKey struct {
```

Comment:
Question - can we put these OF specific definitions into pkg/agent/openflow?

Reply:
I think we can move `MatchKey` but let's wait for other comments.

Reply:
Hello @jianjuns , I attempted to move `MatchKey` but there will be some problems. If I only move this struct and its related methods, it will introduce circle import. And if I move `MatchKey`, `Address*`, `MatchKey`, `PolicyRule`, there will be tons of changes. With next releasing date approaching, I suggest keep it as it is. We can refactor the logic in future commit.

Reply:
Sure. Could we have a follow-up PR to refactor the code?